src/InlineBlockList.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import cx from 'classnames';
+
+export const INLINEBLOCKLIST_CLASS = 'inlineblockList';
+export const INLINEBLOCKLIST_SEPERATED_CLASS = 'inlineblockList--separated';
+
+/**
+ * @module InlineBlockList
+ */
+class InlineBlockList extends React.Component {
+	render() {
+		const {
+			className,
+			items,
+			separator,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			INLINEBLOCKLIST_CLASS,
+			{
+				[INLINEBLOCKLIST_SEPERATED_CLASS]: separator
+			},
+			className
+		);
+
+		return (
+			<ul
+				className={classNames}
+				{...other}>
```

Comment:
minor code style thing - we've been wrapping the closing `>` to the next line for readability